### DISCUSSION TITLE
Bytes type gives Error: Unknown type "Bytes". Did you mean "Date"?

### DISCUSSION THREAD

**Discussion by apsHugh** [prisma/prisma#22428] [Q&A] [answered]
### Question

I want to use the bytea column type on Postgres and am trying to set Prisma to play nicely with it. I have an input model to which I've added a property with the Bytes data type. But when I build it it gives me this error: 

Error: Unknown type "Bytes". Did you mean "Date"?

I am using Prisma version:  4.9.0

This is my model...
```prisma
input CreateDatagridinstanceInput{
  id:  String!
  originalId: String!
  definition:  JSON!
  defCompressed: Bytes
  status:  String!
  parentId: String
  title: String
  customerId: String
  regionId: String
  supplierId: String
  regimeId: String
  gridSchemaId: String
}
```
### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @apsHugh 👋 

Thank you for raising this question.

Prisma [models](https://www.prisma.io/docs/orm/prisma-schema/data-model/models) begin with the `model` keyword followed by the model name. To create a correct model in Prisma, you should do like the code below:
```prisma
model YourModel {
  // ... other fields ...
  defCompressed Bytes?
  // ... other fields ...
}
```
And you would need to ensure that your Prisma schema is set up correctly with the PostgreSQL datasource. Here's an example of how the datasource might look:
```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```
You are receiving an error about an unknown type "Bytes", it could be because you are trying to use Prisma types in a context that expects GraphQL types. Please ensure you are modifying your Prisma file in a `schema.prisma` file.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by apsHugh** [reply]
Yes - you are exactly right. I was trying to use Bytes in the .gql file instead ot eh schema.primsa file. Many thanks!
